mantidproject/mantid
# Review RRO Workflow diagram

After a series of changes the workflow diagram for the ReflectometryReductionOne algorithm does not match with the flow of code. This PR aims to change the workflow diagram so that it is representative of the current code for the algorithm.

**To test:**
- Build Qt-help documentation for ReflectometryReductionOne-v1
- Step through the workflow and see that it represents the flow in the code.

Fixes #15534 



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
